### ISSUE TITLE
Include CO2 forcing and surface albedo feedback from `forcing.f90`

### ISSUE THREAD

**Issue by duncanwp** [climate-analytics-lab/jax-gcm#61]
Speedy (optionally) implements CO2 forcing by gradually ramping up CO2 LW absorption, and it would be good to have a similar option. It also seems to includes a simple surface albedo feedback which it would be nice to have.

It should be simple to include both by porting the relevant parts of `forcing.f90`, most of the rest of that code is redundant though, or taken care of by Dinosaur. This should be simple once #60 is done since it can hook into the same interface.

There's not much point in doing either of the above until we have at least a slab ocean though.
